tilequeue/query/postgres.py

```diff
@@ -67,9 +64,15 @@ def __init__(self, sources, query_generator):
     def __call__(self, zoom, bounds):
         queries = []
         for source in self.sources:
-            if source.start_zoom <= zoom:
-                query = self.query_generator(source.template, bounds, zoom)
-                queries.append(query)
+            template_queries = []
+            for template_spec in source.template_specs:
+                if template_spec.zoom_start <= zoom < template_spec.zoom_stop:
```

Comment:
My personal opinion is that exclusive just reads better here, but that could just be me being used to that coming from a programming context. The ranges in the sql were already exclusive, and imho it's clearer to follow what's going on.

For example, this is typical:

```
zoom_start: 0
zoom_stop: 2

zoom_start: 2
zoom_stop: 5

zoom_start: 5
zoom_stop: 8
```

Spelled inclusively:

```
zoom_start: 0
zoom_stop: 1

zoom_start: 2
zoom_stop: 4

zoom_start: 5
zoom_stop: 7
```

About consistency with the other configuration in the transforms, maybe we should just update it all to be exclusive across the board? Or, maybe we just strictly adhere to a naming convention to clearly delineate which is used. Initially I had been using the term `zoom_until` to mean inclusive, but it looks like that's fallen out of fashion because I don't see it there any more.